src/Compilers/CSharp/Portable/Binder/Binder.ValueChecks.cs

```diff
@@ -796,7 +950,10 @@ private uint GetParameterValEscape(ParameterSymbol parameter)
             }
         }
 
-        private uint GetParameterRefEscape(ParameterSymbol parameter)
+        private static EscapeLevel? GetParameterValEscapeLevel(ParameterSymbol parameter, bool useUpdatedEscapeRules) =>
```

Comment:
>GetParameterValEscapeLevel [](http://example.com/codeflow?start=36&length=26)

`GetParameterValEscapeLevel()` and `GetParameterRefEscapeLevel()` are unusual (compared other escape analysis methods) because they return a subset of values, and return that as an `EscapeLevel?` rather than a `uint`. Given that, and the fact that these are only used in one call-site each, consider inlining the calls.

Reply:
@RikkiGibson suggested I not inline
@cston suggested I inline

:)

Fine with either but at the moment would prefer to keep as is. Happy to change.